ansible_base/authentication/views/ui_auth.py

```diff
@@ -40,7 +40,6 @@ def generate_ui_auth_data():
             response['passwords'].append(
                 {
                     'name': authenticator.name,
-                    'type': authenticator.type,
                 }
```

Comment:
Lets check with the UI team as to how they are using this. We likely don't even need to expose the name here as the logic should be something like `if len(reponse['passwords']: show the password on the login form`. So maybe we could just do something like `response['passwords'].append(True)`. Ideally, if possible, we could just convert this to a boolean but the UI would likely need to make a change for that unless they are already doing something `if response['passwords']`.

Reply:
ui team does not seem to have any usage or intentions to use `response['passwords']`. Removing

Reply:
Forgot that making passwords True would break typing of UIAuth model. Any recommendations there?

Reply:
If they don't use it at all maybe it can be removed entirely?